sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/tests/MiscTests.cs

```diff
@@ -63,6 +63,33 @@ public async Task TokenIssuanceStartObjectModelTest()
             Assert.True(TestHelper.DoesPayloadMatch(Payload.TokenIssuanceStart.ExpectedPayload, httpResponseMessage.Content.ReadAsStringAsync().Result));
         }
 
+        /// <summary>Test the request object to verify the correct HttpStatusCode is respond</summary>
+        [Test]
+        [TestCase(RequestStatusType.Successful, HttpStatusCode.OK)]
+        [TestCase(RequestStatusType.Failed, HttpStatusCode.BadRequest)]
+        [TestCase(RequestStatusType.ValidationError, HttpStatusCode.BadRequest)]
```

Comment:
Did we have a scenario where if there are multiple things wrong with the response, we can aggregate them and throw all together instead of 1 by 1?

Reply:
is this what you're referring to? [Bug 2433233](https://identitydivision.visualstudio.com/Engineering/_workitems/edit/2433233): Return all errors at once